content/docs/3.0/tasks/deploy/introduction.md

```diff
@@ -56,37 +56,48 @@ The **suggested PD** specifications for production are:
 
 * 3+ nodes
 * 4+ cores
-* 8+ GB RAM
-* 200+ GB SSD
+* 8+ GB RAM, with no swap space.
+* 200+ GB Optane, NVMe, or SSD drive
 * 10 Gigabit ethernet (2x preferred)
 
 The **suggested TiKV** specifications for production are:
 
 * 3+ nodes
 * 16+ cores
-* 32+ GB RAM
-* a less than 1.5 TB SSD
+* 32+ GB RAM, with no swap space.
+* 200+ GB Optane, NVMe, or SSD drive (Under 1.5 TB capacity is ideal in our tests)
+* 10 Gigabit ethernet (2x preferred)
 
 
 ## Network requirements
 
+TiKV deployments require **total connectivity of all services**. Each TiKV, PD, and client must be able to reach each all other and advertise the addresses of all other services to new services. This connectivity allows TiKV and PD to replicate and balance data resiliently across the entire deployment.
+
+If the hosts are not already able to reach each other, it is possible accomplish this through a Virtual Local Area Network (VLAN). Speak to your system administrator to explore your options.
+
 TiKV requires the following network port configuration to run. Based on the TiKV deployment in actual environments, the administrator can open relevant ports in the network side and host side.
 
-| Component | Default Port | Description |
-| :--:| :--: | :-- |
-| TiKV | 20160 | the TiKV communication port |
-| PD | 2380 | the inter-node communication port within the PD cluster |
-| Pump | 8250 | the Pump communication port |
-| Drainer | 8249 | the Drainer communication port |
-| Prometheus | 9090 | the communication port for the Prometheus service|
-| Pushgateway | 9091 | the aggregation and report port for TiKV, and PD monitor |
-| Node_exporter | 9100 | the communication port to report the system information of each TiKV cluster node |
-| Blackbox_exporter | 9115 | the `Blackbox_exporter` communication port, used to monitor the ports in the TiKV cluster |
-| Grafana | 3000 | the port for the external Web monitoring service and client (Browser) access|
-| Grafana | 8686 | the `grafana_collector` communication port, used to export the Dashboard as the PDF format |
-| Kafka_exporter | 9308 | the `Kafka_exporter` communication port, used to monitor the binlog Kafka cluster |
+| Component | Default Port | Protocol | Description |
+| :--:| :--: | :--: | :-- |
+| TiKV | 20160 | gRPC | Client (such as Query Layers) port. |
+| TiKV | 20180 | Text | Status port, Prometheus metrics at  `/metrics`. |
+| PD | 2379 | gRPC | The client port, for communication with clients. |
+| PD | 2380 | gRPC | The server port, for communication with TiKV. |
+
+{{< info >}}
+If you are deploying tools alongside TiKV you may need to open or configure other ports. Such as port 3000 on a Grafana service, so that it is accessible.
```

Comment:
```suggestion
If you are deploying tools alongside TiKV, you may need to open or configure other ports, so that they are accessible.  For example, port 3000 for a Grafana service,
```